tests/bsim/bluetooth/host/gatt/caching/test_scripts/_run_test.sh

```diff
@@ -19,3 +19,14 @@ Execute ./bs_2G4_phy_v1 -v=${verbosity_level} -s=${simulation_id} \
     -D=2 -sim_length=60e6 $@
 
 wait_for_background_jobs
+
+Execute ./bs_${BOARD_TS}_tests_bsim_bluetooth_host_gatt_caching_prj_conf_psa_overlay_conf \
```

Comment:
I havent read all comments so this may be justified, so please bear with me.
This is used by 6 tests.
Is there a good reason to double those 6? 
Meaning, is there a good reason why this is done in a shared subscript instead of having its own separate test?
(I just want to be sure we do not start increasing CI time a lot without a good reason).

Reply:
Well, based on the failures I had when I first introduced PSA tests here, I think that 1 single case for the GATT should be enough. I mean, once I got 1 single GATT test passing, then all the others passed as well. I can improve this part

Reply:
No need for the double psa in the sim_id  (sorry that was just a silly mistake of mine), you don't need to change it back either